README.md

```diff
@@ -42,7 +42,7 @@ yarn add web3
 ## Migration Guide
 
 -   [Migration Guide from Web3.js 1.x to 4.x](https://docs.web3js.org/docs/guides/web3_migration_guide)
-    Breaking changes are listed in migration guide and its first step for migrating from Web3.js 1.x to 4.x. If there is any question or discussion feel free to ask in discord, and in case of any bug or new feature request [open issue](https://github.com/web3/web3.js/issues/new) or create a pull request for [contributions](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).
+    Breaking changes are listed in migration guide and its first step for migrating from Web3.js 1.x to 4.x. If there is any question or discussion feel free to ask in [Discord](https://discord.gg/yjyvFRP), and in case of any bug or new feature request [open issue](https://github.com/web3/web3.js/issues/new) or [create a pull request]('https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request') for [contributions](https://github.com/web3/web3.js/blob/4.x/CONTRIBUTIONS.md).
```

Comment:
The Discord link is a good addition, I'm not sure about the Github doc link to creating a PR

Reply:
Ok. Should I delete create PR link?

Reply:
Im in favor of deleting github doc link. I agree with @spacesailor24 people who are creating PRs should already know the process beforehand and if not they could always communicate on the discord or inquire further knowledge.